langchain/chains/base.py

```diff
@@ -115,20 +115,52 @@ def __call__(
         else:
             return {**inputs, **outputs}
 
+
+class SingleInputChain(Chain, BaseModel, ABC):
+    """Base interface for chains that take in a single input."""
+
+    input_key: str = Field(default="input")  #: :meta private:
```

Comment:
can probably just do ` = "input"` instead of the field (unless im missing something)